### PR TITLE
Js file commenting

### PR DESCRIPTION
**Pull request by zmalkmus**
I have added developer comments for each AGVis js file. Each one includes a header that describes the author, date modified, description of the file, and a link to the API documentation. I mostly created header comments for each function per the following standard [https://developer.wordpress.org/coding-standards/inline-documentation-standards/javascript/](url). I also created breaks in logical code sections for readability. Finally, I reformatted the code styling (indentations mostly) to be consistent across all js files.

### CODE REVIEW

File: agvis/static/js/ConfigControl.js [hunk 1]
```diff
@@ -397,11 +473,14 @@ function addSidebarConfig(win, options, sidebar) {
             dt = dt.toUTCString();
 
             document.cookie = `opacity${win.num}=${val};expires=${dt};path=/`;
-
-            win.legend.update();
```
Comment (zmalkmus) [inline agvis/static/js/ConfigControl.js:401]: This change removed a line that was accidentally added when making the legend bar.

**Comment by jinningwang** [comment]
Zack, well done! I believe this can greatly improve the maintainability of AGVis.

The blog link you insert seems to be invalid. Is there any automated tool can do this for your future development? If so, you can save manual efforts in the future.

**Review by jinningwang — APPROVED** [review]

**Comment by zmalkmus** [comment]
> Zack, well done! I believe this can greatly improve the maintainability of AGVis.
> 
> The blog link you insert seems to be invalid. Is there any automated tool can do this for your future development? If so, you can save manual efforts in the future.

Sorry about that, I messed up the link placement. Its fixed now. 

I'm not sure if there are tools for auto commenting in this particular style, but it will be trivial to maintain this commenting style with future development anyways.

**Comment by jinningwang** [comment]
> > Zack, well done! I believe this can greatly improve the maintainability of AGVis.
> > The blog link you insert seems to be invalid. Is there any automated tool can do this for your future development? If so, you can save manual efforts in the future.
> 
> Sorry about that, I messed up the link placement. Its fixed now.
> 
> I'm not sure if there are tools for auto commenting in this particular style, but it will be trivial to maintain this commenting style with future development anyways.

No need to do this for all files in the future, just mark the most important files to your perspective.

**Pull request merged**